redis/src/cmd.rs

```diff
@@ -601,6 +615,38 @@ impl Cmd {
     pub fn is_no_response(&self) -> bool {
         self.no_response
     }
+
+    /// It marks command as triable for client side caching.
+    /// Useful when `CacheMode` is set to `CacheMode::OptIn`
+    #[cfg(feature = "cache-aio")]
+    pub fn cache(&mut self) -> &mut Cmd {
```

Comment:
IMO this should take `client_side_ttl` and remove `cache_with_client_side_ttl`.
Do you think that future iterations might take more arguments? should we create some config struct for per-command caching, in order to maintain forward compatibility.

Reply:
Creating option struct and having one function to take it makes more sense because currently this function also enables for OptIn mode and once OptOut is introduced there will be need for another function.